Frends.HubSpot.UpdateCompany/Frends.HubSpot.UpdateCompany.Tests/.env.example

```
@@ -0,0 +1 @@
+FRENDS_HubSpot_privateAccessToken="example"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)